pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -666,6 +666,15 @@ func GetOpenshiftBootstrapClusterRoles() []rbac.ClusterRole {
 		},
 		{
 			ObjectMeta: metav1.ObjectMeta{
+				Name: NodeConfigReaderRoleName,
```

Comment:
This looks like it should be namespace scoped.  https://github.com/openshift/origin/pull/16517 made it easily possible

Reply:
Lets make this a role since we remove the namespace configuration.